meta-iot2000-example/recipes-support/swupdate/swupdate_%.bbappend

```diff
@@ -1,6 +1,6 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/swupdate:"
 
-SRCREV = "012374087c747f05216a7f002e4b8a34bc142c52"
+#SRCREV = ""
```

Comment:
Looks like a debug left-over. Or, if there is not need for deviation from the upstream selection anymore, just remove the line.

Reply:
Your right, line it not needed anymore.